app/src/main/java/com/github/se/studybuddies/ui/account/CreateAccount.kt

```diff
@@ -71,30 +82,56 @@ fun CreateAccount(userViewModel: UserViewModel, navigationActions: NavigationAct
       }
   val permission = imagePermissionVersion()
 
-  Column(modifier = Modifier.fillMaxSize().testTag("create_account")) {
-    LazyColumn(
-        modifier = Modifier.fillMaxWidth().padding(horizontal = 20.dp, vertical = 20.dp),
-        verticalArrangement = Arrangement.spacedBy(20.dp, Alignment.Top),
-        horizontalAlignment = Alignment.CenterHorizontally) {
-          item {
-            Column(
-                modifier = Modifier.fillMaxWidth().testTag("content"),
-                verticalArrangement = Arrangement.spacedBy(20.dp)) {
-                  Text(stringResource(R.string.you_have_signed_in_with_email, email))
-                  Spacer(modifier = Modifier.padding(20.dp))
-                  AccountFields(usernameState)
-                  Spacer(modifier = Modifier.padding(20.dp))
-                  SetProfilePicture(photoState) {
-                    checkPermission(context, permission, requestPermissionLauncher) {
-                      getContent.launch(imageInput)
+  Scaffold(
+      modifier = Modifier.fillMaxSize().background(White).testTag("create_account"),
+      topBar = {
+        TopNavigationBar(
+            title = { Sub_title(title = stringResource(R.string.create_account)) },
+            navigationIcon = {
+              Icon(
+                  imageVector = Icons.Default.ArrowBack,
+                  contentDescription = "Go back",
+                  modifier =
+                      Modifier.clickable {
+                            FirebaseAuth.getInstance().signOut()
+                            navigationActions.navigateTo(Route.LOGIN)
+                          }
+                          .testTag("go_back_button"))
+            },
+            actions = {})
+      }) { paddingValue ->
+        Column(
+            modifier = Modifier.fillMaxSize(),
+            horizontalAlignment = Alignment.Start,
+            verticalArrangement = Arrangement.Top) {
+              LazyColumn(
```

Comment:
Did we really need to be able to scroll ?
If not, we should remove the Lazycolumn and directly display our elements

Reply:
I followed the AccountSetting formatting to uniformize everything